### ISSUE TITLE
glumpy.app.Window should not depend on the argparse module

### ISSUE THREAD

**Issue by drufat** [glumpy/glumpy#50]
When using PyDev(or PyCharm) as an IDE, it is impossible to launch glumpy windows in the interactive console  using the common Ctrl+Alt+Enter key binding. Whenever one tries to do that, for example by running the code snippet below:

```python
from glumpy import app

window = app.Window()

@window.event
def on_draw(dt):
    window.clear()

app.run()
```
one gets the following error
```
/home/user/Applications/anaconda3/bin/python -u /home/user/Applications/pycharm-4.5.4/helpers/pydev/pydevconsole.py 50818 42698
Python 3.4.3 |Anaconda 2.2.0 (64-bit)| (default, Jun  4 2015, 15:29:08) 
Type "copyright", "credits" or "license" for more information.

IPython 3.1.0 -- An enhanced Interactive Python.
Anaconda is brought to you by Continuum Analytics.
Please check out: http://continuum.io/thanks and https://anaconda.org
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.
PyDev console: using IPython 3.1.0

import sys; print('Python %s on %s' % (sys.version, sys.platform))
sys.path.extend(['/home/user/Source/glumpy', '/home/user/Source/vispy'])

Python 3.4.3 |Anaconda 2.2.0 (64-bit)| (default, Jun  4 2015, 15:29:08) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)] on linux
In[2]: from glumpy import app
window = app.Window()
@window.event
def on_draw(dt):
    window.clear()
app.run()

usage: pydevconsole.py [-h] [--backend {glfw,sdl2,pyside,pyglet,sdl,osxglut}]
                       [--record] [--interactive] [--framerate FRAMERATE]
                       [--display-fps] [--debug] [--size SIZE]
                       [--position POSITION] [--single-buffer] [--stereo]
                       [--vsync VSYNC] [--srgb] [--depth-size DEPTH_SIZE]
                       [--stencil-size STENCIL_SIZE] [--gl-api {GL,ES}]
                       [--gl-profile {none,core,compatibility}]
                       [--gl-version GL_VERSION]
pydevconsole.py: error: unrecognized arguments: 50818 42698

Process finished with exit code 2
```
No such error occurs when launching matplotlib or vispy windows.

The reason for the failure seems to be in the usage of the argparse module. Since glumpy windows can be launched from within the python interpreter, I do not think it is proper to rely on argparse and access command line arguments. In the case above glumpy is trying to access the arguments that were supplied to pydevconsole.py from within which it is being launched, and sicne those arguments were not intended for glumpy it is unable to parse them, we get an exception and a failure to launch any windows.

There are two ways to fix this. 

**Fix 1**
The hard fix would be to change the three ocurrences of the code below in glumpy/app/__init__.py in a way that gets the options without access the command line arguments when launched from within the interprter
```python
options = parser.get_options
```
**Fix 2**
The easier fix is to just change `get_options()` in `glumpy/app/parser.py` so that inability to parse arguments does not result in an failure. This can be achieved by using `parse_known_args` rather than `parse_args`. That ways arguments that cannot be parsed will be ignored. The proposed new get_options() function would then be 

```python
def get_options():
    """ Parse and retrun command line options. """
    options, unknown = get_default().parse_known_args()
    return options
```

**Comment by rougier** [comment]
Make sense. Can you make a PR for the second method (seems to be cleaner)

**Comment by rougier** [comment]
Fixed by #51
